ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -300,10 +317,34 @@ open class BadgeView: UIView {
         }
     }
 
+    private var labelSize: CGSize {
+        let size = label.sizeThatFits(CGSize(width: CGFloat.infinity, height: CGFloat.infinity))
+        let width = UIScreen.main.roundToDevicePixels(size.width)
+        let height = UIScreen.main.roundToDevicePixels(size.height)
+        return CGSize(width: width, height: height)
+    }
+
+    private var customViewPadding: UIEdgeInsets {
+        let getFloat: (_ number: NSNumber?, _ defaultValue: CGFloat) -> CGFloat = { (number, defaultValue) in
+            if let number = number {
+                return CGFloat(truncating: number)
+            }
+            return defaultValue
+        }
+        return UIEdgeInsets(
+            top: getFloat(dataSource?.customViewVerticalPadding, size.verticalPadding),
```

Comment:
Could you store the paddings in constants here so you don't need to repeat the call multiple times?
```
let verticalPadding = size.verticalPadding
let horizontalPadding = size.horizontalPadding
```

Reply:
Fixed